src/Pages/Home/Home.tsx

```diff
@@ -9,17 +9,17 @@ export function Home(): JSX.Element {
         <div>
         <HeroImageOfTheDay/>
         <ul className="roverNav">
-            <RoverNavItem imageUrl={"https://robohash.org/CuriosityRover?bgset=bg1"}
+            <RoverNavItem imageUrl={"https://image.freepik.com/free-vector/group-people-holding-question-mark-icons_53876-64627.jpg"}
                           align={"rightAlign"}
                           altText={"Curiosity Rover"}
                           description={"Curiosity"}
                           destination={"/curiosity"}/>
-            <RoverNavItem imageUrl={"https://robohash.org/OpportunityRover?bgset=bg1"}
+            <RoverNavItem imageUrl={"https://media.npr.org/assets/img/2018/06/15/mom-support_slide-b7832f16b707e5bde84de23c0277f0328d7bd6f9-s800-c85.jpg"}
                           align={"leftAlign"}
                           altText={"Opportunity Rover"}
                           description={"Opportunity"}
                           destination={"/opportunity"}/>
-            <RoverNavItem imageUrl={"https://robohash.org/SpiritRover?bgset=bg1"}
+            <RoverNavItem imageUrl={"https://image.freepik.com/free-vector/teenagers-friends-having-fun-cartoons_18591-52609.jpg"}
```

Comment:
Could we source control copies of these images in our `public` file?

That way we 
a) source control what is important to us
b) don't depend on anyone else.